bin/diagnostic.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer >= 1 && <= 10) {
+   console.log("Your guess needs to be between 1 and 10");
+   break;
+  }
```

Comment:
`(answer >= 1 && answer <= 10)` is checking if a number is between 1 and 10.

It is saying 'if answer is greater than or equal to 1 AND answer is less than or equal to 10'